apps/dapp-console-api/src/routes/auth/AuthRoute.ts

```diff
@@ -76,8 +82,165 @@ export class AuthRoute extends Route {
     },
   )
 
+  public readonly isAttested = 'isAttested' as const
+  public readonly isAttestedController = this.trpc.procedure
+    .input(
+      this.z
+        .object({
+          address: zodEthereumAddress,
+        })
+        .describe('returns whether the address is attested for in any way'),
+    )
+    .query(async (req) => {
+      // Checks if user is Gitcoin Passport attested with a score > 25
+      const gitcoinPassportAttestation = await getGitcoinPassportAttestation(
+        req.input.address,
+      )
+      if (
+        gitcoinPassportAttestation &&
+        gitcoinPassportAttestation.score.profileScore > 25
+      ) {
+        return true
+      }
+
+      // Checks if user is attested for Optimism Faucet access by an allowlisted attester
+      const attestersArray = envVars.FAUCET_ACCESS_ATTESTERS
+
+      if (!attestersArray || attestersArray.length === 0) {
+        return false
+      }
+
+      const tempFaucetAttestation = await getTempFaucetAccessAttestation(
+        req.input.address,
+        attestersArray,
+      )
+
+      if (tempFaucetAttestation) {
+        return true
+      }
+
+      const coinbaseVerificationAttestation =
+        await getCoinbaseVerificationAttestationFromEAS(req.input.address)
+
+      if (coinbaseVerificationAttestation) {
+        return true
+      }
+
+      return false
+    })
+
+  public readonly isCoinbaseVerified = 'isCoinbaseVerified' as const
+  public readonly isCoinbaseVerifiedController = this.trpc.procedure
+    .input(
+      this.z
+        .object({
+          address: zodEthereumAddress,
+        })
+        .describe('returns whether the address is attested for in any way'),
+    )
+    .query(async (req) => {
+      const coinbaseVerificationAttestation =
+        await getCoinbaseVerificationAttestationFromEAS(req.input.address)
+
+      if (coinbaseVerificationAttestation) {
+        return true
+      }
+
+      return false
+    })
```

Comment:
is this needed, or could you just use the `isAttested` endpoint from the frontend?

Reply:
its needed since the query is slightly different from other attestations